src/utilities/algolia.js

```diff
@@ -0,0 +1,139 @@
+import slugify from 'slugify'
+
+export const prefixSearchIndex = index =>
+  `${process.env.GATSBY_ALGOLIA_INDEX_PREFIX}${index}`
+
+const stateQuery = `{
+  states: allCovidStateInfo(
+    filter: { name: { ne: null } }
+  ) {
+    edges {
+      node {
+        objectID: state
+        name
+        state
+      }
+    }
+  }
+}`
+
+const blogPostQuery = `{
+  posts: allContentfulBlogPost {
+    edges {
+      node {
+        objectID: contentful_id
+        title
+        author {
+          name
+        }
+        slug
+        lede
+        publishDate(formatString: "MMMM D, YYYY")
+        body {
+          body
+        }
+      }
+    }
+  }
+}`
+
+const pagesQuery = `{
+  posts: allContentfulPage {
+    edges {
+      node {
+        objectID: contentful_id
+        title
+        slug
+        body {
+          body
+        }
+      }
+    }
+  }
+}`
+
+function splitBodyIntoChunks(baseChunk, firstChunk, bodyChunks) {
```

Comment:
I assume this is based on Agolia limits? Could you add a docstring to clarify?

Reply:
Added some doc to get a better understanding of what it does and why.